### PR TITLE
Add ch.vbs.kataster-belasteter-standorte-militaer

### PR DESCRIPTION
**Pull request by b4l**
Test link: https://mf-chsdi3.dev.bgdi.ch/ltbal_kbs_vbs/shorten/8b36972bb7

### CODE REVIEW

File: chsdi/templates/htmlpopup/kataster_belasteter_standorte_militaer.mako [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    str_output = c['attributes'][untersuchungsmassnahmen].replace('##', '<br />')
+%>
+
+    <tr><td class="cell-left">${_('ch.vbs.kataster-belasteter-standorte-militaer.katasternummer')}</td>                        <td>${c['attributes']['katasternummer'] or '-'}</td></tr>
+    <tr><td class="cell-left">${_('ch.vbs.kataster-belasteter-standorte-militaer.standorttyp')}</td>                           <td>${c['attributes'][standorttyp] or '-'}</td></tr>
+    <tr><td class="cell-left">${_('ch.vbs.kataster-belasteter-standorte-militaer.status_altlv')}</td>                          <td>${c['attributes'][status_altlv] or '-'}</td></tr>
+    <tr><td class="cell-left" valign="top">${_('ch.vbs.kataster-belasteter-standorte-militaer.untersuchungsmassnahmen')}</td>  <td>${_(str_output)|n}</td></tr> 
+    <tr><td class="cell-left">${_('ch.vbs.kataster-belasteter-standorte-militaer.url_kbs_auszug')}</td>                        <td><a href="${c['attributes']['url_kbs_auszug'] or '-'}" target="_blank">${_('link')}</a></td></tr>
+    <tr><td>&nbsp;</td>                                                                                                       <td>&nbsp;</td></tr>
```
Comment (rebert) [inline chsdi/templates/htmlpopup/kataster_belasteter_standorte_militaer.mako:18]: what does the last line serve for? &nbsp;?

File: chsdi/templates/htmlpopup/kataster_belasteter_standorte_militaer.mako [hunk 2]
```diff
@@ -0,0 +1,20 @@
+<%inherit file="base.mako"/>
+
+<%def name="table_body(c, lang)">
+
+<%
+    lang = lang if lang in ('fr', 'it') else 'de'
+    standorttyp = 'standorttyp_%s' % lang
+    status_altlv = 'status_altlv_%s' % lang
+    untersuchungsmassnahmen = 'untersuchungsmassnahmen_%s' % lang
+    str_output = c['attributes'][untersuchungsmassnahmen].replace('##', '<br />')
```
Comment (rebert) [inline chsdi/templates/htmlpopup/kataster_belasteter_standorte_militaer.mako:10]: top

**Review by rebert — APPROVED** [review]
what does the last line in the table serve for? &nbsp;

File: chsdi/templates/htmlpopup/kataster_belasteter_standorte_militaer.mako (hunk 1 above)
Comment (b4l) [reply chsdi/templates/htmlpopup/kataster_belasteter_standorte_militaer.mako]: I don't know so i removed it.

**Comment by ltclm** [comment]
@rebert in the meantime i have checked the codebuild tests in aws console, tests are passing.

**Pull request merged**